addons/addon-base/packages/serverless-go-build-tools/index.js

```diff
@@ -79,6 +92,45 @@ class GoBuildTools {
       return;
     }
 
+    // Test out Go command
+    try {
+      execSync('go version');
+    } catch (error) {
+      // error: Go is not installed
+      // Ask if they want to deploy Windows workspaces, as they will not work properly without Go lang
+      this.cli.log(
+        messagePrefix,
+        'If you want to deploy Windows workspaces within Service Workbench, you must have Go lang installed before deployment. This is recommended.',
+      );
+      let validAnswer = false;
+      while (!validAnswer) {
+        /* eslint-disable no-await-in-loop */
+        const response = await this.readLineAsync(
+          `${messagePrefix} Do you plan on configuring and using Windows workspaces? (y/N) `,
+        );
+        /* eslint-enable no-await-in-loop */
+        if (response.match(/^(y|Y|yes|Yes|YES)$/)) {
+          // If Windows workspaces are desired, fail build
+          this.cli.warn(messagePrefix, 'Failing building due to lack of Go lang. Please install Go lang.');
+          readline.close();
+          validAnswer = true;
+          throw new Error('Go lang not installed');
+        } else if (response.match(/^(n|N|no|No|NO)$/)) {
+          // If Windows workspaces are not desired, log another warning and do not fail build
+          this.cli.warn(
+            messagePrefix,
+            'Any Windows workspaces deployed will not function properly. You have been warned.',
```

Comment:
suggestion: 'Any Windows workspaces deployed will not function properly _until Go is installed and SWB is redeployed_. You have been warned.'

Reply:
Got it!